app/metrics.yaml

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+fxr_metrics:
```

Comment:
This does not make it too much different from `metrics`. Please see our [naming guidelines](https://mozilla.github.io/glean/book/user/adding-new-metrics.html#naming-things).

A good category name here would be `distribution` and a good metric name could be `channel_name`.